src/main/kotlin/org/jetbrains/intellij/dependency/PluginDependencyManager.kt

```diff
@@ -148,7 +142,8 @@ open class PluginDependencyManager @Inject constructor(
                 addArtifact(IntellijIvyArtifact.createDirectoryDependency(it, configuration.name, baseDir, groupId))
             }
             ideaDependency?.sources?.takeIf { plugin.builtin }?.let {
-                val artifact = IntellijIvyArtifact(it, "ideaIC", "jar", "sources", "sources")
+                val name = if (isDependencyOnPyCharm(ideaDependency)) "pycharmPC" else "ideaIC"
```

Comment:
I think it's ok, but for even better experience it makes sense to download pycharm sources and attach them to Python plugin dependency even if `ideaDependency` is IntelliJ